boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -165,6 +163,30 @@ pinout.  Power to the ESP32 board should be provided by a USB cable.
 | IO15       | TDO       |
 +------------+-----------+
 
+Once the device is connected, you should be able to connect with (for
+a DevKitC board, replace with esp32-wrover.cfg for WROVER):
+
+.. code-block:: console
+
+    cd ~/esp/openocd-esp32
+    sudo src/openocd -f interface/ftdi/flyswatter2.cfg -c 'set ESP32_ONLYCPU 1' -c 'set ESP32_RTOS none' -f board/esp-wroom-32.cfg -s tcl
```

Comment:
`sudo` might not be necessary if `udev` rules are set appropriately.  It's good practice to discourage people from running commands as superuser if possible.

Reply:
Yeah, that was a mistaken cut & paste.  Needless to say I don't bother with udev.  It's always wrong, and one more thing that always gets forgotten when moving or reinstalling systems.